lusSTR/workflows/strs.smk

```diff
@@ -76,13 +74,24 @@ def parse_sample_details(filename):
 def create_log(log):
     now = datetime.now()
     dt = now.strftime("%m%d%Y_%H_%M_%S")
-    shell("mkdir -p logs/{dt}/input/")
-    shell("cp '{log}' logs/{dt}/")
-    if os.path.isdir(input_name):
-        shell("cp '{input_name}'/*.* logs/{dt}/input/")
+    system = os.name
+    if system == "nt":
+        shell("md logs\\{dt}\\Input\\")
+        shell('copy "{log}" logs\\{dt}\\')
+        shell("copy config.yaml logs\\{dt}\\")
+        new_file = input_name.replace("/", "\\")
+        if os.path.isdir(input_name):
+            shell('xcopy "{new_file}" logs\\{dt}\\Input')
+        else:
+            shell('copy "{new_file}" logs\\{dt}\\Input\\')
     else:
-        shell("cp '{input_name}' logs/{dt}/input/")
-    shell("cp config.yaml logs/{dt}/")
+        shell("mkdir -p logs/{dt}/input/")
+        shell("cp '{log}' logs/{dt}/")
+        if os.path.isdir(input_name):
+            shell("cp '{input_name}'/*.* logs/{dt}/input/")
+        else:
+            shell("cp '{input_name}' logs/{dt}/input/")
+        shell("cp config.yaml logs/{dt}/")
```

Comment:
@standage I've added this to account for the different code required for a Mac/linux vs. windows os... is this kosher or not a good strategy to deal with it? This is the only difference between the Mac and windows version.